src/awx_plugins/credentials/_managed_types/terraform.py

```diff
@@ -0,0 +1,45 @@
+"""HCP Terraform managed credential type."""
+# mypy: disable-error-code="assignment, misc"
+
+from awx_plugins.interfaces._temporary_private_api import (  # noqa: WPS436
+    ManagedCredentialType,
+)
+from awx_plugins.interfaces._temporary_private_django_api import (  # noqa: WPS436
+    gettext_noop,
+)
+
+from awx_plugins.credentials.injectors import (
```

Comment:
Use relative imports internally, instead of relying on the top-level package name.
```suggestion
from ..injectors import (
```